glue/app/qt/application.py

```diff
@@ -205,6 +208,10 @@ def __init__(self, data_collection=None, session=None):
         Application.__init__(self, data_collection=data_collection,
                              session=session)
 
+        # Create eventFilter and pull in any keybindings from keyboard_shortcuts.py
```

Comment:
I would remove the part about keyboard_shortcuts as there may be other shortcuts registered with ``keybindings``.